### ISSUE TITLE
Error loading webview | The goals panel isn't rendered

### ISSUE THREAD

**Issue by yannl35133** [coq-community/vscoq#464]
The panel to see goals is not rendered, making the extension close to useless.
> Error loading webview: Error: Could not register service worker: InvalidStateError: Failed to register a ServiceWorker: The document is in an invalid state.
The panel to see goals is not rendered, making the extension close to useless.

Visual Studio Code v 1.78.0 for Linux (Manjaro) through Flatpak
Also happens in Codium v 1.78.1 for Linux (Manjaro, vscodium-bin in the AUR) (my usual setup), but the error seems to be silenced there

Does not happen in Codium v 1.78.1 for Windows (WSL remote for Coq)
Did not happen in Codium v 1.77.3 for Linux (Manjaro, vscodium-bin in the AUR)

How to reproduce :
- Create a .v file
- Write a goal (`Goal True.`)
- Step in (Alt+Down usually)

**Comment by 4ever2** [comment]
Looking at https://github.com/VSCodium/vscodium/issues it seems like a lot of extensions using webviews are broken in VSCodium 1.78.1

**Comment by yannl35133** [comment]
The exact versions :
VSCode :
```
Version: 1.78.0
Commit: 252e5463d60e63238250799aef7375787f68b4ee
Date: 2023-05-03T20:09:46.685Z
Electron: 22.4.8
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Linux x64 6.3.0-1-MANJARO
Sandboxed: Yes
```
VsCoq: v0.3.8 (only extension installed)
Coq(top) : `The Coq Proof Assistant, version 8.16.1 compiled with OCaml 4.14.1`

**Comment by thery** [comment]
Ok the only diff with mine is that you have `Sandboxed: Yes` I have `No`

**Comment by MevenBertrand** [comment]
I seem to be having a similar issue, with the following version:
```
Version: 1.78.1
Release: 23130
Commit: 748cf23d39091308b3b0d65ac43f0d784709fdcc
Date: 2023-05-10T01:49:13.634Z
Electron: 22.3.5
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Linux x64 5.19.0-41-generic snap
Sandboxed: No
```

**Comment by huynhtrankhanh** [comment]
Looks like there's nothing we can do. I'm closing this issue.

**Comment by yannl35133** [comment]
On the VSCodium side, it seems this was fixed with version 1.78.2
On the VSCode side, this is also fixed for me now, but I don't understand as nothing changed (except my Linux kernel maybe, but this seems largely irrelevant).

Anyway, it's all good for me now, thanks
